pkg/bench/tpcc/subprocess_utils_test.go

```diff
@@ -33,7 +34,9 @@ func (c *cmd) exec(env cmdEnv, args ...string) *exec.Cmd {
 	cmd.Env = os.Environ()
 	cmd.Env = append(cmd.Env, fmt.Sprintf("%s=t", internalTestEnvVar))
 	cmd.Env = append(cmd.Env, env.toStrings()...)
-	return cmd
+	var buf bytes.Buffer
+	cmd.Stdout = &buf
```

Comment:
sometimes the interesting bits are in stdout (maybe not the case here, in which case disregard). If you wanted to capture both,

```go
type synchronizedBuffer struct {
    buf  bytes.Buffer
    lock sync.Mutex
}

func (s *synchronizedBuffer) Write(p []byte) (n int, err error) {
    s.lock.Lock()
    defer s.lock.Unlock()
    return s.buf.Write(p)
}
```

can be used for a combined `cmd.Stdout` and `cmd.Stdout`.

Reply:
Great idea, done.